desc/compute/_field.py

```diff
@@ -2488,7 +2488,8 @@ def _B_sub_zeta_rz(params, transforms, profiles, data, **kwargs):
     label="\\langle |\\mathbf{B}| \\rangle_{axis}",
     units="T",
     units_long="Tesla",
-    description="Average magnitude of magnetic field on the magnetic axis",
+    description="Average magnitude of magnetic field on the flux surface "
```

Comment:
Can you mention somewhere (on the given grid) or something? Sounds a bit weird as is, like we don't evaluate things on axis or something